.gitignore

```diff
@@ -0,0 +1,3 @@
+.env
```

Comment:
This file is a configuration file for Git. It tells Git which files we do not want to track and commit. This is useful for not bloating our repository with large folders like `node_modules` which are recreated on the deployment server anyways.

It's also used to hide files that contain sensitive information like our buddy `.env`.